config/local-setup/dns-provider/inmemory/kustomization.yaml

```diff
@@ -10,4 +10,4 @@ secretGenerator:
   - name: dns-provider-credentials
     type: "kuadrant.io/inmemory"
     literals:
-      - INMEM_INIT_ZONES=kuadrant.local
+      - INMEM_INIT_ZONES=e2e.hcpapps.net
```

Comment:
Why change this?

Reply:
to align with the rest of the providers. It is a "taste" change not a "make it work" one

Reply:
It doesn't make sense to me to change this to a domain that we actually use and is managed by route53. Not sure how this aligns with anything, its just a different domain, and one that is set to something obviously not meant for actual external use to avoid confusion.